src/allencell_ml_segmenter/thresholding/thresholding_view.py

```diff
@@ -85,14 +85,15 @@ def __init__(
         layout.addWidget(self._title, alignment=Qt.AlignmentFlag.AlignHCenter)
 
         # selecting input image
-        self._file_input_widget: FileInputWidget = FileInputWidget(
-            self._file_input_model,
-            self._viewer,
-            self._input_files_service,
-            include_channel_selection=False,
+        self._prediction_result_input_widget: PredictionResultListWidget = (
+            PredictionResultListWidget(
+                self._file_input_model,
+                self._viewer,
+                self._input_files_service,
+            )
         )
-        self._file_input_widget.setObjectName("fileInput")
-        layout.addWidget(self._file_input_widget)
+        self._prediction_result_input_widget.setObjectName("fileInput")
```

Comment:
is `setObjectName` a method of a superclass?  Does it set a label?  An input field type?  Is this something that our component could handle internally?